Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -409,40 +408,16 @@ def bottle_formula(f, args:)
           tab.write
         end
 
-        keg.find do |file|
-          # Set the times for reproducible bottles.
-          if file.symlink?
-            # Need to make symlink permissions consistent on macOS and Linux
-            # TODO: Refactor and move to extend/os
-            File.lchmod 0777, file if OS.mac? # rubocop:disable Homebrew/MoveToExtendOS
-            File.lutime(tab.source_modified_time, tab.source_modified_time, file)
-          else
-            file.utime(tab.source_modified_time, tab.source_modified_time)
-          end
-        end
-
         cd cellar do
           sudo_purge
           # Tar then gzip for reproducible bottles.
-          tar, tar_args = setup_tar_and_args!(args)
-          safe_system tar, "--create", "--numeric-owner",
-                      *tar_args,
-                      "--file", tar_path, "#{f.name}/#{f.pkg_version}"
-          sudo_purge
-          # Set more times for reproducible bottles.
-          tar_path.utime(tab.source_modified_time, tab.source_modified_time)
-          relocatable_tar_path = "#{f}-bottle.tar"
```

Comment:
This filename also affects the checksum I think and I don't see any replacement for that here.

Reply:
@Bo98 reverted now.